sleuth/sleuth/enrich_tls.py

```diff
@@ -203,20 +268,37 @@ def enrich_tls(flow, kwargs):
         else:
             scs = None
 
+
         if 'server_cert' in tls:
             certs = list()
             for x in tls['server_cert']:
                 tmp = dict()
-                tmp['signature_algorithm'] = x['signature_algorithm']
-                tmp['signature_key_size'] = x['signature_key_size']
+                tmp['sig_alg'] = x['signature_algorithm']
+                tmp['sig_key_size'] = x['signature_key_size']
+                tmp['issuer'] = x['issuer']
                 certs.append(tmp)
         else:
             certs = None
-
+        
         seclevel_policy = Policy(kwargs["policy_file"], kwargs["failure_threshold"])
 
+        certs_seclevel, certs_concerns = get_certs_seclevel(seclevel_policy, certs, kwargs["unknowns"])
```

Comment:
You could make a global ignore_unknowns variable and then assign kwargs["unknowns"] to it. Then you can remove the parameter from these 2 functions to make it cleaner.